sdks/java/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
Can you add these to .gitignore so that they are never added whenever someone clicks the button "enable something something with Gradle" in Intellij

Reply:
obsolete